openlibrary/templates/account/create.html

```diff
@@ -33,7 +33,7 @@ <h1>$_("Sign Up")</h1>
 $if ctx.user:
     $def user_link(): <a href="$ctx.user.key">$ctx.user.displayname</a>
     <p>$:_("You are already logged into Open Library as %(user)s.", user=str(user_link()))</p>
-    <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'logout\'].submit()">log out</a> and start the signup process afresh.')</p>
+    <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'logout\'].trigger('submit')">log out</a> and start the signup process afresh.')</p>
```

Comment:
```suggestion
    <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'logout\'].trigger(\'submit\')">log out</a> and start the signup process afresh.')</p>
```

Single quotes surrounding `submit` must be escaped, or the `create.html` page will throw an error.

Reply:
Disregard previous suggestion.  This should be rolled back to the original code, as `document.forms.logout` is not a jQuery object.

Reply:
dispatchEvent is the native equivalent